convert2rhel/utils.py

```diff
@@ -287,11 +287,13 @@ def get_executable_name():
     return os.path.basename(inspect.stack()[-1][1])
 
 
+def is_root():
+    return os.geteuid() == 0
```

Comment:
I don't think this function is adding too much value to the `require_root` function. The old way was simple enough to understand. 

It was quicker to simply look at the if statement and figure out how the condition would be met or not, here now we have to hop into a function, get a boolean result from it and negate that same result inside the `require_root`. 

I would return the check to `os.geteuid() != 0` instead.